planner/core/point_get_plan.go

```diff
@@ -172,17 +180,37 @@ func (p *PointGetPlan) SetOutputNames(names types.NameSlice) {
 	p.outputNames = names
 }
 
+// GetCost returns cost of the PointGetPlan.
+func (p *PointGetPlan) GetCost(cols []*expression.Column) float64 {
```

Comment:
Can we simply return a constant here?

Reply:
Why? You mean when we can get the point-get plan, other plans should not be considered?

Reply:
Yes